Content.Server/Body/Systems/RespiratorSystem.cs

```diff
@@ -354,15 +359,11 @@ private void TakeSuffocationDamage(Entity<RespiratorComponent> ent)
 
         _damageableSys.TryChangeDamage(ent, ent.Comp.Damage, interruptsDoAfters: false);
 
-        if (ent.Comp.SuffocationCycles >= ent.Comp.SuffocationCycleThreshold)
-        {
-            // TODO: This is not going work with multiple different lungs, if that ever becomes a possibility
-            var organs = _bodySystem.GetBodyOrganEntityComps<LungComponent>((ent, null));
-            foreach (var entity in organs)
-            {
-                _alertsSystem.ShowAlert(ent, entity.Comp1.Alert);
-            }
-        }
+        if (ent.Comp.SuffocationCycles < ent.Comp.SuffocationCycleThreshold)
```

Comment:
is there a reason the damage starts _before_ suffocation? I know it's legacy but it seems weird.